examples/dishwasher-app/silabs/src/ElectricalSensorManager.cpp

```diff
@@ -196,12 +196,12 @@ void ElectricalSensorManager::UpdateEPMAttributes(OperationalStateEnum state)
 {
     if (gEPMDelegate)
     {
-        uint16_t updateState       = to_underlying(state);
-        uint16_t ERROR_STATE_INDEX = ArraySize(kAttributes) - 1;
+        uint8_t updateState = to_underlying(state);
```

Comment:
In the future, I would recommend `auto` in this situation.  That prevents accidental narrowing (which would also be prevented if this code were built with `-Wconversion`, but I bet it's not).